src/main/java/org/opentripplanner/graph_builder/module/osm/GermanyWayPropertySetSource.java

```diff
@@ -0,0 +1,119 @@
+package org.opentripplanner.graph_builder.module.osm;
+
+import static org.opentripplanner.graph_builder.module.osm.WayPropertySetSource.DrivingDirection.RIGHT_HAND_DRIVE;
+
+import org.opentripplanner.routing.edgetype.StreetTraversalPermission;
+
+/**
+ * OSM way properties for German roads Speed limits where adjusted to German regulation and some
+ * bike safety settings tweaked, especially including tracktype's grade and preference of bicycle
+ * networks.
+ *
+ * @see WayPropertySetSource
+ * @see DefaultWayPropertySetSource
+ */
+public class GermanyWayPropertySetSource implements WayPropertySetSource {
+
+    @Override
+    public void populateProperties(WayPropertySet props) {
+        // Replace existing matching properties as the logic is that the first statement registered takes precedence over later statements
+
+        /*
+         * Automobile speeds in Germany. General speed limit is 50kph in settlements, 100kph outside settlements.
+         * For motorways, there (currently still) is no limit. Nevertheless 120kph is assumed to reflect varying
+         * traffic conditions
+         *
+         */
```

Comment:
Code style: 

4.8.6.1 On multi-line `/* ... */` comments we do not begin the intermediate lines with asterisks `*`.

I personally avoid using C-style comments mixed with code, the only place I use it is to make sections in a class like /* private methods */.